Packs/Bitbucket/Integrations/Bitbucket/Bitbucket_test.py

```diff
@@ -0,0 +1,594 @@
+
+
+import json
+import io
+
+import pytest
+from Bitbucket import Client
+
+
+@pytest.fixture
+def bitbucket_client():
+    return Client(workspace='workspace',
+                  server_url='server_url',
+                  auth=(),
+                  proxy=False,
+                  verify=False,
+                  repository='repository')
+
+
+def util_load_json(path):
+    with io.open(path, mode='r', encoding='utf-8') as f:
+        return json.loads(f.read())
+
+
+def test_get_paged_results(mocker, bitbucket_client):
+    """
+        Given:
+            - A http response and a limit to the list
+        When:
+            - running a command with pagination needed
+        Then:
+            - return a list with all the results after pagination
+        """
+    from Bitbucket import get_paged_results
+    response1 = util_load_json('test_data/commands_test_data.json').get('get_paged_results').get('response1')
+    response2 = util_load_json('test_data/commands_test_data.json').get('get_paged_results').get('response2')
```

Comment:
Do the load in the tests just once at the beginning of each test and then use it.
json_response = util_load_json(''path")
and then use the response to get what you need 
For all the tests